crates/egui/src/context.rs

```diff
@@ -2123,12 +2123,15 @@ impl Context {
 
         self.write(|ctx| {
             let tessellation_options = ctx.memory.options.tessellation_options;
-            let texture_atlas = ctx
-                .fonts
-                .get(&pixels_per_point.into())
-                .expect("tessellate called with a different pixels_per_point than the font atlas was created with. \
-                         You should use egui::FullOutput::pixels_per_point when tessellating.")
-                .texture_atlas();
+            let texture_atlas = match ctx.fonts.get(&pixels_per_point.into()) {
+                Some(fonts) => fonts.texture_atlas(),
+                None => {
+                    ctx.fonts.iter().next()
```

Comment:
This will fall back to a texture atlas for the wrong pixels_per_point, which will likely look bad. I'd rather figure out the root cause of the bug than mask it :/

Reply:
> This will fall back to a texture atlas for the wrong pixels_per_point, which will likely look bad. The existing check is there to help users that write egui integration to find bugs in their code.
> 
> I'd rather figure out the root cause of the bug than mask it :/

Don't worry, the texture atlas will be read for good pixel_per_point right away.
However, if you don't do this, a panic will occur before the texture atlas for a good pixel_per_point is read.
How to reproduce the panic is the same as the sequence shown above.

This also eliminates the root cause of the bug.
The root cause of another bug was fixed in #4128, so I didn't find it.